src/sentry/search/events/builder/errors.py

```diff
@@ -126,6 +127,16 @@ def get_snql_query(self) -> Request:
             tenant_ids=self.tenant_ids,
         )
 
+    def add_conditions(self, conditions: list[Condition]) -> None:
```

Comment:
Oh interesting

So - we don't need to do anything in the subscription processor?
Was entity only necessary for the errors??